ratpack-pac4j/src/test/groovy/ratpack/pac4j/openid/OpenIdRpSpec.groovy

```diff
@@ -46,6 +48,17 @@ class OpenIdRpSpec extends RatpackGroovyDslSpec {
     }
 
     handlers {
+      prefix 'prefix', {
+        configurePac4j delegate
+      }
+
+      configurePac4j delegate
+    }
+
+  }
+
+  GroovyChain configurePac4j(GroovyChain chain) {
```

Comment:
Yes - totally agree with GroovyChainAction here. Will update.
The noauth handler is under both root and prefixed paths so needs to be added twice.